training/Train.py

```diff
@@ -80,6 +70,12 @@ def main():
             Utils.csvwrite('logs/trainloss.csv',\
                            [epoch,epoch_train_loss.average()])
 
```

Comment:
I would add a del train_dataset and del train_data_loader line here. Idk if having multiple instances of the same hdf5 file open will cause problems.